src/routes/blog/post/scan-receipts-with-appwrite-functions/+page.markdoc

```diff
@@ -122,11 +122,11 @@ import os
 def main(context):
     # Why not try the Appwrite SDK?
     #
+    # Set API key and set project
```

Comment:
The order of the code is project id first and then api key so it would make most sense to match that.